Makefile

```diff
@@ -104,7 +104,8 @@ start-indexer: ##
 	cargo run -p indexer --release -- --home-dir ~/.near/localnet run --da-contract-ids da.test.near --rollup-ids 2 --rmq-address "amqp://127.0.0.1:5672"
 
 start-test-relayer: ##
-	go run relayer/cmd/main.go --rpc-url ws://127.0.0.1:8546 --da-account-id da.test.near
+	export CGO_LDFLAGS="-L ./relayer/libs"
```

Comment:
It's more complicated than that [here](https://github.com/taco-paco/rollup-data-availability/blob/e6354bde000692b84a4dde1581babe2619b5b00e/gopkg/da-rpc/near_darwin.go#L5). The setup is platform dependent

And it also seems that it will override those flags

Reply:
This shall be defined on a particular package level

Reply:
From doc [here](https://pkg.go.dev/cmd/cgo)

> When building, the CGO_CFLAGS, CGO_CPPFLAGS, CGO_CXXFLAGS, CGO_FFLAGS and CGO_LDFLAGS environment variables are added to the flags derived from these directives. Package-specific flags should be set using the directives, not the environment variables, so that builds work in unmodified environments. Flags obtained from environment variables are not subject to the security limitations described above.

 this my statement is incorrect

> Also that seems dangerous since other libraries may also have their own CGO_LDFLAGS definitions and may fail to compile

Nevertheless documentation still points out the same thing:

> Package-specific flags should be set using the directives, not the environment variables, so that builds work in unmodified environments.

Reply:
May I suggest another angle,

How about instead of using `da-rpc-sys` you straight up use the `http-api`. Yes you would need to run a sidecar with the relayer, but the seperation would be very clean.

Reply:
> Also that seems dangerous since other libraries may also have their own CGO_LDFLAGS definitions and may fail to compile

To avoid overriding the environment we could do something like `CGO_LDFLAGS="-L ./relayer/libs go run ...", but the fact that the flags are platform dependent makes things a bit tricky.

Nevertheless, lets follow the docs: we'll set package specific flags through directives.

Reply:
Solved by reusing existing `CGO_LDFLAGS`. Closing discussion to merge.